server/lock/leader.go

```diff
@@ -0,0 +1,41 @@
+package lock
+
+import (
+	"time"
+
+	"github.com/fleetdm/fleet/v4/server"
+)
+
+var owner = ""
+
+func init() {
+	var err error
+	owner, err = server.GenerateRandomText(64)
+	if err != nil {
+		panic(err)
+	}
+}
```

Comment:
We've generally tried to avoid `init` functions where possible because of arguments like (https://peter.bourgon.org/blog/2017/06/09/theory-of-modern-go.html). I'm open to discussing whether this is a standard we should try to keep. If we agree, I can document it in the coding conventions I am building (https://github.com/fleetdm/fleet/issues/1352).

Reply:
Fair point, will read soon, but I'm sure I agree.

New try, even simpler, let me know what you think.